metron-platform/metron-parsers/src/main/java/org/apache/metron/parsers/topology/ParserTopologyBuilder.java

```diff
@@ -132,7 +132,14 @@ public static TopologyBuilder build(String zookeeperUrl,
     if(securityProtocol.isPresent()) {
       kafkaSpoutConfigOptions.putIfAbsent("security.protocol", securityProtocol.get());
     }
-    return SimpleStormKafkaBuilder.create(inputTopic, zkQuorum, Arrays.asList("value"), kafkaSpoutConfigOptions);
+    return SimpleStormKafkaBuilder.create( inputTopic
```

Comment:
I think that prefixing the names of the metadata with `metadata.` would correct that, I think.